examples/dotnet/CsharpAspServer/Program.cs

```diff
@@ -45,33 +45,28 @@ public static void Main(string[] args)
             context.Response.Redirect("index.html");
             return Task.CompletedTask;
         });
-        app.MapGet("/language/{lang:required}", (string lang, IServerSentEventService sseService) => sseService.MergeFragments($"""<span id="language">{lang}</span>"""));
-        app.MapGet("/patch", async (IServerSentEventService sseService, IDatastarSignals signals) =>
+        app.MapGet("/title/{title:required}", (string title, IServerSentEventService sseService) => sseService.MergeFragments($"""<span id="title">C# + {title}</span>"""));
+        app.MapGet("/check_more_examples", () => Task.CompletedTask); // do nothing
+        app.MapGet("/patch", async (IServerSentEventService sseService, ISignals dsStore) =>
         {
-            DataSignals signals = (signals as DataSignals) ?? throw new InvalidCastException("Unknown DataSignals passed");
-            DataSignals mergeSignals = new() { Output = $"Patched Output: {signals.Input}" };
+            Signals signals = (dsStore as Signals) ?? throw new InvalidCastException("Unknown Datastore passed");
```

Comment:
Should `dsStore` and `Datastore` be renamed? We no longer use `store` anywhere in the library or docs.

Reply:
Thanks for catching. Will take a look.

Reply:
Thanks for catching. Will take a look.

Reply:
Corrected, didn't see any other "store"s.